test/regress/cli/regress0/seq/proj-issue586-nth-update-no-fact.smt2

```diff
@@ -0,0 +1,7 @@
+; EXPECT: sat
+(set-logic ALL)
+(set-option :seq-array eager)
+(declare-const x String)
+(declare-const x5 (Bag String))
```

Comment:
Are bags and strings really necessary to trigger the bug on main and to exhibit the fix? It would be better to have a test (perhaps in addition) that is a purely sequences benchmark, or at least with less theories involved. Is that possible?